cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/server.go

```diff
@@ -15,41 +15,40 @@ package main
 import (
 	"context"
 	"fmt"
+	"io/ioutil"
+	"net/http"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
 
+	"github.com/ghodss/yaml"
 	corev1 "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/core/packages/v1alpha1"
 	"github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/plugins/fluxv2/packages/v1alpha1"
 	"k8s.io/client-go/rest"
+	helmrepo "k8s.io/helm/pkg/repo"
+	log "k8s.io/klog/v2"
 )
 
-// Server implements the kapp-controller packages v1alpha1 interface.
+const (
+	// see docs at https://fluxcd.io/docs/components/source/
+	fluxGroup            = "source.toolkit.fluxcd.io"
+	fluxVersion          = "v1beta1"
+	fluxHelmRepoResource = "helmrepositories"
```

Comment:
I've updated my PR at #2857 with consistent constants, thanks.

Reply:
> Nice work Greg! Take a look at the example tests I've added in #2857 , though you'll need a bit extra for the `index.yaml` (should be able to use the httptest for that). See whether you want to include the tests in this PR or add them as a subsequent PR. I don't mind either way.

Thank you for kind words. Yes, subsequent PR. I will definitely be adding unit tests